src/views/nads/form/utils/utils.ts

```diff
@@ -0,0 +1,129 @@
+import { adjectives, animals, uniqueNamesGenerator } from 'unique-names-generator';
```

Comment:
please go over all this file... :)

Reply:
Please be more specific like what you did in buildConfig :)

Reply:
i think u understood according to the new version :)